docker/docker-compose.override.yml

```diff
@@ -9,20 +9,24 @@ services:
   collector:
     environment:
       - COLLECTOR_LOG_LEVEL=${COLLECTOR_LOG_LEVEL:-INFO}
+      - DATABASE_HOST=localhost
     build:
       context: ../components
       dockerfile: collector/Dockerfile
     depends_on:
       - testdata
   notifier:
     environment:
+      - DATABASE_HOST=localhost
       - NOTIFIER_LOG_LEVEL=${NOTIFIER_LOG_LEVEL:-INFO}
     build:
       context: ../components
       dockerfile: notifier/Dockerfile
   api_server:
     environment:
       - API_SERVER_LOG_LEVEL=${API_SERVER_LOG_LEVEL:-INFO}
+      - DATABASE_HOST=localhost
+      - LDAP_URL=ldap://localhost:389
```

Comment:
Should this not be the ldap container instead of localhost?

Reply:
Yeah, that was a mistake. Removed.